api/authentication.thrift

```diff
@@ -28,4 +29,10 @@ service codeCheckerAuthentication {
   // performs logout action for the user (must be called from the corresponding valid session)
   bool destroySession()
              throws (1: shared.RequestFailed requestError)
+
+  // returns currently logged in user within the active session
+  // returns empty string if the session is not active
```

Comment:
Maybe for better type safety, this should return `None` instead of `""` if no user is present.